src/kdc/do_as_req.c

```diff
@@ -546,6 +546,10 @@ process_as_req(krb5_kdc_req *request, krb5_data *req_pkt,
     /* Seed the audit trail with the request ID and basic information. */
     kau_as_req(kdc_context, TRUE, au_state);
 
+    if ((errcode = krb5_timeofday(kdc_context, &state->kdc_time)))
```

Comment:
Nit: put the assignment on its own line outside the conditional.  (I recognize you're just copying it from below.)